test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -37,6 +37,10 @@ const (
 )
 
 var (
+	defaultWorkspaceEnvVars = map[string]string{
+		"PULUMI_SKIP_UPDATE_CHECK": "true",
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**